Makefile

```diff
@@ -54,7 +54,7 @@ build-image:
 	docker build ${BUILDARGS} -t ${AGENT_IMAGE}:${AGENT_TAG} .
 
 proto: build-image
-	docker run -ti -v ${PWD}:/go/src/github.com/kata-containers/agent ${AGENT_IMAGE}:${AGENT_TAG} ./hack/update-generated-agent-proto.sh
+	docker run -i -v ${PWD}:/go/src/github.com/kata-containers/agent ${AGENT_IMAGE}:${AGENT_TAG} ./hack/update-generated-agent-proto.sh
```

Comment:
Sorry I fail to see how this **checks** if pb.go regenerated right. Could you please enlighten me?

Reply:
I think the `-t` has probably been removed here since Jenkins doesn't provide a tty to the process, so this change is required to allow `make proto` to work under the CI.

But yes, it's not checking so much as "ensuring the regeneration target doesn't fail".

See https://github.com/kata-containers/agent/pull/350#issuecomment-417590652.

Reply:
Yes, different version of protoc will generate quite different pb.go files. I think we all use v3.5 luckily.